Sources/PackageGraph/Pubgrub.swift

```diff
@@ -1103,14 +1123,15 @@ public final class PubgrubDependencyResolver<
                             terms.append(Term(not: dep.identifier, dep.requirement))
                         }
                         return Incompatibility(terms, root: root!, cause: .dependency(package: candidate.package))
-                }
+                    }
                 depIncompatibilities.forEach { add($0, location: .decisionMaking) }
 
+                let tmp = PartialSolution(assignments: solution.assignments)
+                tmp.decide(candidate.package, atExactVersion: version)
+                // Check if this decision would result in a conflict when added.
+                // If so, we try the next earlier version instead.
+                #warning("Why is this depIncompatibilities and not incompatibilities[candidate.package]?")
```

Comment:
@kiliankoe can you remove this warning and make a new PR?

Reply:
nvm: https://github.com/apple/swift-package-manager/pull/1978

Reply:
Ah sry, this shouldn't have been staged 🤦‍♂️